docs/quest_template.md

```diff
@@ -223,3 +223,399 @@ Contains all basic definitions of available quests.
 [105]: #VerifiedBuild
 
 **Description of the fields**
+
+### ID
+
+The quest ID. This column is the Primary Key for the Table. Each quest ID must be unique!
+
+### QuestType
+
+Accepted values: 0, 1, 2 or 3. Their meaning is described in table below.
```

Comment:
useless info, only have the table below.

Reply:
It was the information of the original file, my intention was not to edit it, only to update it. But mostly in terms of structure. Since I don't know much about English, it's hard for me to write and describe some things. In Spanish I could do it.